docs/labs/10-Timer.md

```diff
@@ -128,9 +138,12 @@ To start the timer so that it repeatedly calls `actionPerformed`,
 you must tell that particular timer to start,
 by calling its `start` method.
 So to start the Timer,
-you'll see a line like this shortly after creating a `Timer` object.
+it would look like this,
+with the method shortly after creating a `Timer` object with `this`
+refering to the class that it is in.
 
 ```java
+Timer someTimerVar = new Timer(1000, this);
```

Comment:
I would want further context here, next to the comment, because some students would want to copy this, and so we want to make sure they think of the timer creation and the start as separate steps.  So either a comment or something to let them know this is optional or can be done in a constructor or beforehand.